src/_angular/url-field/dtv-url-field.js

```diff
@@ -73,6 +93,10 @@
               scope.invalidType = "url";
             }
 
+            if (isValid) {
+              testImage();
```

Comment:
@stulees Since `testImage` is asynchronous, I thought it best to just go ahead and return the current `isValid` status, and then update it later when the function returns. Otherwise, there will be a delay in either removing or showing the error. Worst case scenario now is that the error will disappear and then reappear.